README.md

```diff
@@ -1 +1,109 @@
-# Postgres-Database--based-on-a-diagram
\ No newline at end of file
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [Prerequisites](#prerequisites)
+  - [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 [Postgres-Database--based-on-a-diagram] <a name="about-project"></a>
+
+
+**[Postgres-Database--based-on-a-diagram]** IS a relational database created as the data structure for a clinic. that contains tables of patients, treatments, invoices, medical_histories, and invoice_items information.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Creates a Relational Database]**
+- **[Designed from a schema design]**
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+1. SQL shell.
+2. Code editor.
+3. PostgreSQL.
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
+
+### Setup
+
+Clone this repository to your desired folder:
+it clone https://github.com/danielochuba/Postgres-Database--based-on-a-diagram.git
+
+## 👥 Author <a name="authors"></a>
+
+👤 **Daniel Ochuba**
+- GitHub: [@danielochuba](https://github.com/danielochuba)
+- Twitter: [@ochuba_daniel](https://twitter.com/ochuba_daniel)
+- LinkedIn: [Daniel Ochuba](https://www.linkedin.com/in/daniel-ochuba-ugochukwu)
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Add a UI]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/danielochuba/Postgres-Database--based-on-a-diagram/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project Kindly give it a star ⭐.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse for giving me the opportunity to learn and grow as a developer and also I like to thank my family, they are all my support. 🌟
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ Required ] I think it would be better if you fix this link on your readme file, currently, it leads to a `404` page not found error. A broken link on your readme file can be considered as a sign of being unprofessional, please see the example code below for an idea of how to fix this.

## Example code
```sh
This project is [MIT](./LICENSE) licensed.
```
## Current error
![Screenshot from 2023-08-11 18-16-52](https://github.com/danielochuba/Postgres-Database--based-on-a-diagram/assets/16392046/8b341b1d-1d8c-48ac-a5bf-99674c7f20dc)